dotcom-rendering/src/web/components/FrontCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { ArticleDesign } from '@guardian/libs';
+import type { Props as CardProps } from './Card/Card';
+import { Card } from './Card/Card';
+
+type Props = {
+	trail: TrailType;
+} & Partial<CardProps>;
+
+/**
+ * A wrapper around the normal Card component providing sensible defaults for Cards on front containers.
+ *
+ * Any prop used by the original Card can be used in FrontCard to override the defaults.
+ *
+ * Note: Below parameters are not an exhaustive list of params used by FrontCard, rather they are a list of
+ * commonly overridden params to make it easier for a developer to know if they actually need to override these values.
+ *
+ * @param {SmallHeadlineSize} [headlineSize="medium"] - Defaults to "medium"
+ * @param {ImagePositionType} [imagePosition="top"] - Defaults to "top"
+ * @param {ImagePositionType} [imagePositionOnMobile="left"] - Defaults to "left"
+ * @param {ImageSizeType} [imageSize="medium"] - Defaults to "medium"
+ * @param {DCRSupportingContent[]} [supportingContent=undefined] - Defaults to undefined, set to trail.supportingContent if you want this card to show sublinks.
+ */
+export const FrontCard = (props: Props) => {
+	const { trail } = props;
+	const defaultProps: CardProps = {
+		linkTo: trail.url,
+		format: trail.format,
+		headlineText: trail.headline,
+		byline: trail.byline,
+		showByline: trail.showByline,
+		showQuotes:
+			trail.format.design === ArticleDesign.Comment ||
+			trail.format.design === ArticleDesign.Letter,
```

Comment:
Not something that should be done here but this PR makes it clear that this logic could and should live inside the `Card` component itself